### PR TITLE
Update WHITEPAPER.md

### PR DESCRIPTION
**Pull request by shamoons**

### CODE REVIEW

File: WHITEPAPER.md [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
+- `relay node` - a computer in the network that is relaying information between 2 parties
+- `peer` - a `relay node` that either a `visitor` or `relay node` is connected to
+- `coin` - a unit of monetary value. Details TBD
+
+
+# 2. Decentralized Network Discovery
+
+Each `node` in the network is connected (by default) to 10 `peers`. To boostrap the list, similar to the <cite>[Bitcoin P2P bootstrapping mechanism][4]</cite>, Bridgenet will first try to connect to peers that it has connected to before, saved in its local database. If no cached peers can be connected to, it will query a DNS seed to get a list of active peers. If none of those are reachable, it will fallback to a hardcoded peer address. Once a peer is reached, it will be queried to find additional peers.
+
+Each node will send a heartbeat packet (of 10MB) to measure `throughput` periodically. Any peers that have a throughput slower than 1 standard deviation away from the average throughput will be dropped and new peers will be sought.
```
Comment (kfaria) [inline WHITEPAPER.md]: Should we let the peers know they've been dropped to give them the opportunity to fix their bandwidth ?

File: WHITEPAPER.md [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+- `bandwidth` - the data transfer capacity of a computer network in bits per second (Bps)
+- `throughput` - the actual amount of data that is successfully sent/received over the communication link, measured in bits per second (Bps)
+- `relay node` - a computer in the network that is relaying information between 2 parties
+- `peer` - a `relay node` that either a `visitor` or `relay node` is connected to
+- `coin` - a unit of monetary value. Details TBD
+
+
+# 2. Decentralized Network Discovery
+
+Each `node` in the network is connected (by default) to 10 `peers`. To boostrap the list, similar to the <cite>[Bitcoin P2P bootstrapping mechanism][4]</cite>, Bridgenet will first try to connect to peers that it has connected to before, saved in its local database. If no cached peers can be connected to, it will query a DNS seed to get a list of active peers. If none of those are reachable, it will fallback to a hardcoded peer address. Once a peer is reached, it will be queried to find additional peers.
```
Comment (kfaria) [inline WHITEPAPER.md:31]: Will the hardcoded peer address be one of our machines ?

File: WHITEPAPER.md (hunk 1 above)
Comment (shamoons) [reply WHITEPAPER.md]: Hmmm - I’m not 100% sure, but I think they’ll know, right?

If `A` has peers, `B`, `C`, etc, that means at least one of `B`’s peers is `A`, right? So when `A` drops `B`, then when `B` sends a heartbeat to `A`, it’ll error.

What do you think @kfaria ?

File: WHITEPAPER.md (hunk 2 above)
Comment (shamoons) [reply WHITEPAPER.md:31]: I guess the hardcoded values will be nodes that we put up initially? Ideally, we never get to that level of hard coding. Should I change any text here to reflect that?

File: WHITEPAPER.md (hunk 1 above)
Comment (shamoons) [reply WHITEPAPER.md]: @kfaria - do you think I should include this `mutual peer concept in the whitepaper?

**Review by sahmed4322 — APPROVED** [review]

File: WHITEPAPER.md (hunk 2 above)
Comment (kfaria) [reply WHITEPAPER.md:31]: We should just to cover the bases of sound specifications.

**Review by kfaria — APPROVED** [review]

**Pull request merged**